### PR TITLE
Update `Beatmap Spotlights` with references. Add `Beatmap Spotlights/Chart Assembly Team`

### PR DESCRIPTION
**Pull request by TicClick**
closes https://github.com/ppy/osu-wiki/issues/4685

I'd like to keep non-public references for historical value, otherwise it is all documented and backed up scarcely

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by Walavouchey** [comment]
ctrl f for `izi`, `ize` and `iza` and it shows where american spellings have been used

File: wiki/Beatmap_Spotlights/Chart_Assembly_Team/en.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Over the time, there have been several attempts to revitalise the team and increase quality of ranking charts, including two application rounds in 2013[^cat-recruitment-2013] and 2014[^cat-recruitment-2014] and extensive discussions[^cat-hiatus] [^cat-crisis] about possible changes to the team format and responsibilities.
+
+On March 18, 2015, the CAT was dissolved[^cat-death] due to the complete overhaul of ranking charts, which included community-sourced selection of maps.
+
+## References
+
+[^cat-birth]: [(Restricted) Forum post by Derekku (2011-08-29) in "MAT Lounge"](https://osu.ppy.sh/community/forums/posts/1004797)
+[^cat-recruitment-2013]: [Forum thread by Makar (2013-05-19) "2013 NewCAT Applications"](https://osu.ppy.sh/community/forums/topics/133248)
+[^cat-hiatus]: [(Restricted) Forum thread by Ephemeral (2013-09-17) "Ranking chats overhaul"](https://osu.ppy.sh/community/forums/topics/155384)
```
Comment (TPGPL) [inline wiki/Beatmap_Spotlights/Chart_Assembly_Team/en.md:22]: might be a typo

**Review by TPGPL — APPROVED** [review]
looks fine otherwise

File: wiki/Beatmap_Spotlights/Chart_Assembly_Team/en.md (hunk 1 above)
Comment (TicClick) [reply wiki/Beatmap_Spotlights/Chart_Assembly_Team/en.md]: which it is

**Review by TPGPL — APPROVED** [review]

**Pull request merged**